Themes/default/index.template.php

```diff
@@ -87,6 +90,7 @@ function template_init()
  */
 function template_html_above()
 {
+	Theme::loadCSSFile('https://use.fontawesome.com/releases/v6.1.2/css/all.css', array('external' => true));
```

Comment:
Use short arrays

Reply:
This should probably be included by SMF itself if the default theme is including it?

Reply:
> This should probably be included by SMF itself if the default theme is including it?

Yes it should.

Reply:
> This should probably be included by SMF itself if the default theme is including it?

???

@jdarwood007 is saying that you should do this:
```suggestion
	Theme::loadCSSFile('https://use.fontawesome.com/releases/v6.1.2/css/all.css', ['external' => true]);
```

Reply:
I haven't made a commit though, I'm talking about this file being included in the index.template.php

Reply:
@Sesquipedalian is correct in what I was saying.  The SMF code itself is now using short array syntax.  I'm guessing our templates will need to follow that and other syntax changes (We appear to have adopted PSR-12 style with some minor variations).

Reply:
Oh, I think I see now. @DiegoAndresCortes, if I now understand correctly, what you're saying is that this line shouldn't be here at all, but rather it should happen in Sources somewhere. Is that right?

Reply:
Yes that is correct.
The original plan is to allow loading the file/fonts locally too, using a constant variable for the version like jQuery.